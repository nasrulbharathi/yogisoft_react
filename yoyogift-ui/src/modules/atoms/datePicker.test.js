import React from "react";
import { shallow } from "enzyme";
import { DatePickers } from "./datePicker";

describe("DatePickers ", () => {
  it("should be rendered", () => {
    let p = {
      classes: { container: "c" },
    };
    const wrapper = shallow(<DatePickers {...p} />);
    expect(wrapper).toMatchSnapshot();
  });
  it("should be defined", () => {
    expect(DatePickers).toBeDefined();
  });
  it("should render ", () => {
    const props = {
      classes: {},
    };
    const wrapper = shallow(<DatePickers {...props} />);
    expect(wrapper.find("form").length).toBe(1);
  });
});
