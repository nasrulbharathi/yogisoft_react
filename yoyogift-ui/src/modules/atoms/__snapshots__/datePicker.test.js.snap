// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DatePickers  should be rendered 1`] = `ShallowWrapper {}`;

exports[`should render DatePickers  should be defined 1`] = `ShallowWrapper {}`;

exports[`should render DatePickers 1`] = `ShallowWrapper {}`;
